### PR TITLE
Some tiny things about Wiki Import

### PR DESCRIPTION
**Pull request by polinasand**
This PR for `type` initialization (by user in import.html) and some issues with decoding strings.

### CODE REVIEW

File: src/main/webapp/import.html [hunk 1]
```diff
@@ -7,6 +7,33 @@
 <body>
     <div>
         <input name="input" type="text" id="input-text"/>
+        <select id="type">
```
Comment (bprusak) [inline src/main/webapp/import.html:10]: Is this particular list based on some recipes you've already gathered and want to import? Or did you just make it from memory?

File: pom.xml [hunk 2]
```diff
@@ -83,7 +83,14 @@
         <version>5.2.8.RELEASE</version>
         <scope>core</scope>
     </dependency>
-      
+
+    <!-- https://mvnrepository.com/artifact/commons-io/commons-io -->
+    <dependency>
+        <groupId>commons-io</groupId>
```
Comment (bprusak) [inline pom.xml:89]: Please don't add the library if there's no code using it.
Maybe you forgot to add Java code to this pull request?

**Review by bprusak — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 2 above)
Comment (polinasand) [reply pom.xml:89]: I forgot to add this change to previous PR.
I should add it separately from this changes, I think.

File: src/main/webapp/import.html (hunk 1 above)
Comment (polinasand) [reply src/main/webapp/import.html:10]: This list based on https://en.wikibooks.org/wiki/Category:Recipes_by_type list of types, excluding some repeated types or not so popular.

File: src/main/webapp/search-meal-script.js [hunk 3]
```diff
@@ -46,7 +46,7 @@ function createMealElement(dish) {
   const aElement = document.createElement("a");
   aElement.setAttribute("href", `meal.html?id=${id}`);
   const insideDivElement = document.createElement("div");
-  insideDivElement.appendChild(createElementByTag(title, "b"));
+  insideDivElement.appendChild(createElementByTag(capitalizeFirst(title), "b"));
```
Comment (grenlayk) [inline src/main/webapp/search-meal-script.js:49]: I guess I found where problem with encoding appears.
I guess we will need to add `encodingCheck()` here too as I did it in `meal-page-script.js`. 
I made PR to add this func to `script.js` file.

**Pull request merged**